lib/public/views/Runs/Details/index.js

```diff
@@ -86,37 +82,42 @@ const runDetails = (model) => {
         };
 
         return [
-            h('h2.mv2', { onremove: () => model.runs.clearRun() }, `Run #${data.payload.runNumber}`),
-            h('.flex-row.mv2', [
-                h('button.btn.btn-primary.w-15.mh2#create', {
-                    onclick: () => model.router.go(`/?page=log-create&id=${data.payload.runNumber}`),
-                }, 'Add Log Entries'),
-                h('label.form-check-label.f5.w-11.mh2', 'Add log entries to the run'),
-            ]),
-            h('.flex-row.mv2', [
-                h('button.btn.btn-success.w-15.mh2', {
-                    onclick: () => model.runs.updateRunTags(),
-                    title: 'This action will overwrite existing tags',
-                    disabled: !model.runs.selectedTagsChanged(),
-                }, 'Update Tags'),
-                h('label.form-check-label.f5.w-15.mh2', { for: 'tags' }, [
-                    'Add one or multiple tags to the run. ',
-                    '(CTRL + click to select multiple tags).',
+            h('.flex-row', [
+                h('.flex.w-100', [
+                    h('h2.mv2', { onremove: () => model.runs.clearRun() }, `Run #${data.payload.runNumber}`),
+                    h('.flex-row.mv2', [
+                        h('button.btn.btn-primary.w-15.mh2#create', {
+                            onclick: () => model.router.go(`/?page=log-create&id=${data.payload.runNumber}`),
+                        }, 'Add Log Entries'),
+                        h('label.form-check-label.f5.w-11.mh2', 'Add log entries to the run'),
+                    ]),
+                    h('.flex-row.mv2', [
+                        h('button.btn.btn-success.w-15.mh2', {
+                            onclick: () => model.runs.updateRunTags(),
+                            title: 'This action will overwrite existing tags',
+                            disabled: !model.runs.selectedTagsChanged(),
+                        }, 'Update Tags'),
+                        h('label.form-check-label.f5.w-15.mh2', { for: 'tags' }, [
```

Comment:
Moreover, the labels use only `15%` of the space because before they were alone on the entire row. 
I believe they should now be updated together with the buttons. 
Maybe something similar to `w-30` for the buttons to keep the same size and `w-70` for the labels to have the text on one line

Reply:
Fixed